app/client/src/components/pages/Community.Routes.CommunityTabs.js

```diff
@@ -451,9 +453,15 @@ function CommunityTabs() {
         <div css={locationTextStyles}>
           <p css={addressStyles}>{address}</p>
 
-          {watershed && (
+          {watershed.name && (
             <p css={watershedStyles}>
-              <span>WATERSHED:</span> {watershed} ({huc12})
+              <span>WATERSHED:</span> {watershed.name} ({huc12})
+            </p>
+          )}
+          {watershed.areasqkm && watershed.areaacres && (
+            <p css={watershedStyles}>
+              <span>SIZE:</span> {formatNumber(watershed.areaacres)} acres /{' '}
+              {formatNumber(watershed.areasqkm, 2)} km<sup>2</sup>
```

Comment:
I think we should be consistent with the area. Either we should show both acreas and sqkm in popups or this should be updated to only show sqkm.

Reply:
I agree, @coobr01 suggested the former for the summary info, so I'll go ahead and change the popup to match.